packages/block-editor/README.md

```diff
@@ -71,6 +71,10 @@ registerCoreBlocks();
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
+<a name="AlignmentHookSettingsProvider" href="#AlignmentHookSettingsProvider">#</a> **AlignmentHookSettingsProvider**
+
+Undocumented declaration.
```

Comment:
This will be properly documented if we are fine with this approach. But I would prefer to have some feedback on the general mechanism before documenting and polishing it.